packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -55,12 +55,8 @@
 
   /**
    * This component renders a list of suggestions coming from the user queries history.
-   *
-   * @remarks
```

Comment:
Why did you change this? Just asking out of curiosity 😅

Reply:
@LuisMartinez15 it didn't get generated as documentation (if you check [the current docs page](https://docs.empathy.co/develop-empathy-platform/ui-reference/components/history-queries/x-components.history-queries.html), you can see it's not there). I checked the other components applying `baseSuggestions` and they didn't use @remarks, so I thought it was also more consistent to leave it the same way.